### PR TITLE
added README and Json postman

### PR DESCRIPTION
**Pull request by keda-vlad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Bookstore API: Enhancing Your Bookstore Experience
+
+Welcome to the Bookstore API project! Our powerful and secure application, developed using Java Spring Boot, aims to enrich online book purchases for all book enthusiasts. We have combined various features and advanced technologies to provide users with a secure, comprehensive and convenient e-commerce platform. Below, we will introduce you to the key aspects and functional capabilities of our application.
+
+## Features:
+
+### Security:
+
+In our application, we use JWT tokens to ensure secure authentication and authorization. This guarantees that only verified users can access sensitive features.
```
Comment (andrii-hoienko) [inline README.md]:
```suggestion
Bookstore uses JWT tokens to ensure secure authentication and authorization. This guarantees that only verified users can access sensitive features.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Functionality:
+
+In our API, functionality is divided for Users and Admin:
+
+| Controller           | User                                                     | Admin                      |
+|-----------------------|----------------------------------------------------------|------------------------------------|
+| BookController        | Book search with filtering, pagination and specification | Read, update, delete functionality |
+| CategoryController    | Category and books by categories search                  | Read, update, delete functionality |
+| AuthenticationController | Registration, authentication                             | Ability to change user roles |
```
Comment (andrii-hoienko) [inline README.md]: Please align table

File: README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+   **Issue 3: Exception Handling**
+    - Solution: We developed a global exception handler and specific exceptions to improve error reporting.
+
+# Conclusion:
+
+   The Bookstore API provides a robust foundation for building a book-selling platform. We welcome contributions from the community to enhance the Bookstore project. Whether you want to fix a bug, improve an existing feature, or propose a new one, your contributions are valuable to us.
+
+   ### Upcoming Expected Updates:
+
+   We plan to add user recommendation functionality.
```
Comment (andrii-hoienko) [inline README.md]: Who we? I'm alone here

File: README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+## Contact Us:
+
+   For any inquiries or suggestions, please email us at vladyslav.keda@gmail.com.
```
Comment (andrii-hoienko) [inline README.md:111]: Please record loom video

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

File: README.md (hunk 2 above)
Comment (keda-vlad) [reply README.md]: It's align in preview mode
image

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**